app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,81 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@CrossOrigin
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    private final Logger logger = LoggerFactory.getLogger(TaskController.class);
+    private final Map<Long, Task> tasks = new HashMap<>();
+    private Long newTaskId = 1L;
+
+    @GetMapping
+    public ResponseEntity<List<Task>> list() {
+        final var tasksList = new ArrayList<>(tasks.values());
+        return new ResponseEntity<>(tasksList, HttpStatus.OK);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Task> get(@PathVariable("id") final Long id) {
+        final var task = tasks.get(id);
+        if (task == null) {
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+        }
+        return new ResponseEntity<>(task, HttpStatus.OK);
+    }
+
+    @PostMapping
+    public ResponseEntity<Task> create(@RequestBody Task task) {
+        if (task.getTitle().isBlank()) {
+            logger.debug("task: {}", task);
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+        }
+        task.setId(newTaskId++);

```

Comment:
값을 수정하기 전의 값을 주고, 그 다음에 값을 증가시키는 것 같습니다.
이것은 읽는 사람에게 있어서 헷갈릴 수 있기 때문에 분리해서 써주는게 좋을 것 같습니다

Reply:
```java
private Long newTaskId = 0L;

newTaskId += 1;
task.setId(newTaskId);
```

혹시 위와 같이 분리하라는 말씀이실까요?

Reply:
```suggestion
        newTaskId += 1;
        task.setId(newTaskId);
```

혹시 위와 같이 분리하라는 말씀이실까요?

Reply:
네 ㅎㅎ 맞습니다!